py-rattler/LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
Maybe you can turn this into a symlink to the file at the root of the repo? So we dont duplicate this?

Reply:
i could do this, but is there any possibility that py-rattler might me moved to its own repo?

Reply:
personally Id like to keep them together to make sure the two code bases stays as much in sync as possible.